templates/base_index.html

```diff
@@ -112,24 +111,44 @@ <h3><a href="/desktop">Fast, secure and simple, Ubuntu powers millions of PCs wo
     {% block footer_content %}
       {# include "ubuntu/takeovers/_footer_homepage.html" #}
     {% endblock footer_content %}
+    <script>
+        var detectLanguage = function () {
+            var lang = window.navigator.languages ? window.navigator.languages[0] : null; 
+            lang = lang || window.navigator.language ||
+            window.navigator.browserLanguage || window.navigator.userLanguage;
 
+            if (lang && lang.length) {
+                return lang;
+            } else {
+                return null
+            }
+        };
+    </script>
```

Comment:
I assume this was added by rebase'ing on master. This should be done in the future in a separate PR but ok this time.